pkg/operation/botanist/migration.go

```diff
@@ -58,6 +60,50 @@ func (b *Botanist) runParallelTaskForEachComponent(ctx context.Context, componen
 	return flow.Parallel(fns...)(ctx)
 }
 
+// IsCopyOfBackupsRequired check if etcd backups need to be copied between seeds.
+func (b *Botanist) IsCopyOfBackupsRequired(ctx context.Context) (bool, error) {
+	// TODO: check if we can't trigger backup copy for shoots that were migrated before this was implemented and still have a backupentry
+	// in the backupbucket of a different seed. Could be done with a task annotation.
+	if b.Seed.GetInfo().Spec.Backup == nil || !b.isRestorePhase() {
+		return false, nil
+	}
+
+	// First we check if etcd-main etcd resource has been deployed. This is only true if backups have been copied.
+	if _, err := b.Shoot.Components.ControlPlane.EtcdMain.Get(ctx); client.IgnoreNotFound(err) != nil {
+		return false, err
+	} else if err == nil {
+		return false, nil
+	}
+
+	backupEntry, err := b.Shoot.Components.BackupEntry.Get(ctx)
+	if err != nil {
+		return false, err
+	}
+
+	// If the shoot's backupentry has not been switched to the new seed's backup bucket, then backup copying has not been started yet.
+	// And source backup entry has not been created.
+	if backupEntry.Spec.BucketName != string(b.Seed.GetInfo().UID) {
+		return true, nil
+	}
+
+	// TODO: Still have to figure out what to do if the shoot has already been migrated and we want to migrate back ot the original seed
+	// In that case the shoot's backupentry will already point to the proper bucket and no copying has to be done and no error should be
+	// returned in that case
```

Comment:
Or perhaps you are referring to the existing backup entry due to the non-zero `deletionGracePeriodHours`, see https://github.com/gardener/gardener/pull/4894#issuecomment-965285325? But then, backups should be copied also in this case. I am not really sure what could be done to avoid this error, let's discuss it.

Reply:
We agreed to consider this error as non-critical as it's something we see rather as a corner case in production.